golang/cmd/kafka-state-detector/anomalyKafka.go

```diff
@@ -54,7 +54,7 @@ func CloseAnomalyKafka() {
 	zap.S().Infof("[Anomaly]Closing Kafka Consumer")
 	err := AnomalyKafkaConsumer.Close()
 	if err != nil {
-		panic("Failed do close AnomalyKafkaConsumer client !")
+		zap.S().Fatal(err)
```

Comment:
don't we want to have the detailed error message here? "Failed do close AnomalyKafkaConsumer client !